conduct.rst

```diff
@@ -0,0 +1,252 @@
+to half-participate in a conversation.
+This isn't to say you shouldn't help,
+offer advice, or join conversation.
+On the contrary, we encourage all those things.
+Rather, it means that when you want to
+help out or work with others,
+you should fully engage
+and not just interject sporadically.
+If you want to show support
+for an idea presented in a thread,
+consider using a simple thumbs up emoji reaction
+rather than stating the same point a different way.
+If you feel that there is a different approach
+that could be taken,
+but you do not have the time or motivation
+to implement that approach yourself,
+make a note to yourself to explore that approach
+when you have time
+instead of putting that burden on someone else.
+
+No subtle -isms
+---------------
+
+Subtle -isms, also called
+`microaggressions <http://geekfeminism.wikia.com/wiki/Microaggressions>`_,
+are small things that make others feel uncomfortable.
+For example, saying "it's so easy my grandmother could do it" is a subtle -ism,
+as it is both subtly sexist and ageist.
+The "subtle" in "subtle -isms" means that it's probably
+not obvious to everyone right away what was wrong with the comment,
+even to people in the group otherwise affected by the comment.
+And, even though they are subtle,
+might seem insignificant,
+and are often unintentional,
+a steady stream of them compounds
+to make people in under-represented groups feel less welcome.
+
+Time
+====
+
+Be respectful of other people:
+their time, their location,
+and other considerations of a distributed team.
+
+Be on time to meetings.
+Read the meeting agenda ahead of time,
+and if something requires your input
+and you cannot make the meeting,
+tell the meeting organizer.
+
+Resist the temptation
+to have local-only meetings
+when remote teammates should be involved—take them to the forum,
+a group hangout, Github issue, etc. as appropriate.
+
+Keep in mind that what you write once
+will be read by several people.
+Writing a short message means people
+can understand the conversation as efficiently as possible.
+When a long explanation is necessary,
+consider adding a summary.
+Try to bring new arguments to a conversation
+so that each message adds something unique to the thread,
+keeping in mind that the rest of the thread
+sill contains the other messages that have already been made.
+
+Try to stay on topic,
+especially in discussions that are already fairly large.
+If a discussion brings to mind some related
+but separate issue,
+please start a separate discussion for it.
+Everyone that is participating in an existing discussion
+will naturally wish to respond to your comment.
+If it is a separate discussion,
+each person can choose to join that discussion
+or spend their time some other way.
+
+Giving and receiving feedback
+=============================
+
+Give constructive, not critical feedback.
+Feedback is negatively critical
+when it surfaces something wrong with someone or something they produced,
```

Comment:
I would re-write that sentence as "Feedback is negatively critical when it focuses on something wrong with someone"